docs/content/preview/explore/linear-scalability/_index.md

```diff
@@ -53,25 +53,25 @@ To get a better idea of the effort and time you will need to scale your systems,
 
 Data distribution is critical in scaling. In YugabyteDB, data is [split (sharded)](./data-distribution/) into tablets. A tablet is effectively a piece of a table and these tablets are placed on various nodes. The mapping of a row of a table to a tablet is deterministic and the system knows exactly which tablet holds a specific row.
 
-{{<tip>}}
+{{<lead link="./data-distribution/#tablet-splitting">}}
```

Comment:
same thing here.. not sure if we want to have leads or tip in cases where there multiple links.